sass/style.scss

```diff
@@ -1,6 +1,6 @@
-@import 'colors';
-@import 'typography';
-@import 'icons.data.svg';
+@import "colors";
+@import "typography";
+@import "icons.data.svg";
 
```

Comment:
oops sorry for all the linting changes on the `style.scss` file, was trying to avoid. But I think they are fine -- can re-work the commits to exclude formatting changes if preferred

Reply:
fine by me